.expeditor/config.yml

```diff
@@ -1,57 +1,40 @@
 project:
-  alias: omnitruck
+  alias: omnitruck-k8s
 
 slack:
-  notify_channel:
-    - ops-notify
-    - releng-notify
-    - omnitruck
-
+  notify_channel: releng-notify
+  
 github:
   delete_branch_on_merge: true
+  major_bump_labels:
+    - "Expeditor: Bump Version Major"
+  minor_bump_labels:
+    - "Expeditor: Bump Version Minor"
 
-# These are our Buildkite pipelines where deploys take place
-pipelines:
-  - verify:
-      description: Pull Request validation tests
-  - habitat/build:
-      description: Build and publish Habitat artifacts and export Docker containers
-  - deploy/acceptance:
-      description: Deploy changes to kubernetes.chef.co pods
-      definition: .expeditor/deploy.pipeline.yml
-      env:
-        - ENVIRONMENT: acceptance
-        - APP: omnitruck
-  - deploy/production:
-      description: Deploy changes to kubernetes.chef.co pods
-      definition: .expeditor/deploy.pipeline.yml
-      env:
-        - ENVIRONMENT: production
-        - APP: omnitruck
-
-# These actions are taken when `/expeditor promote` is run from Slack
-promote:
-  actions:
-    - built_in:promote_habitat_packages
-    - trigger_pipeline:deploy/production:
-        only_if_conditions:
-          - value_one: "{{target_channel}}"
-            operator: equals
-            value_two: stable
-  channels:
-    - acceptance
-    - stable
 
-merge_actions:
-  - trigger_pipeline:habitat/build:
-      post_commit: true
-      ignore_labels:
-        - "Habitat: Skip Build"
-        - "Expeditor: Skip All"
+# These are our Buildkite pipelines where deploys take place
+pipelines: 
+  - verify
+  - docker/build
 
 subscriptions:
-  # These actions are taken, in order they are specified, after the Habitat packages are successfully built
-  - workload: buildkite_hab_build_group_published:{{agent_id}}:*
+  - workload: pull_request_merged:{{github_repo}}:{{release_branch}}:*
+    actions:
+      - built_in:bump_version:
+          ignore_labels:
+            - "Expeditor: Skip Version Bump"
+            - "Expeditor: Skip All"
+      - bash:.expeditor/update_version.sh:
+          only_if: built_in:bump_version
+      - built_in:update_changelog:
+          ignore_labels:
+            - "Expeditor: Skip Changelog"
+            - "Expeditor: Skip All"
+      - trigger_pipeline:docker/build:
+          only_if_modified: Dockerfile
+          ignore_labels:
+            - "Expeditor: Skip All"
+            - "Expeditor: Skip Docker"
+  - workload: project_promoted:{{github_repo}}:{{release_branch}}:*
```

Comment:
You only needed to change `{{agent_id}}` for the `pull_request_merged` workload. For `project_promoted` you'll want to continue using `{{agent_id}}`. 

The reason I had you change project promoted is because workloads are scoped. The pull request is scoped to you your GitHub repo (`{{github_repo}}` aka `chef/omnitruck`) and your release branch (`{{release_branch}}` aka `kubernetes-migration`). In the past, it just so happened that the form of these two values (`chef/omnitruck:kubernetes-migration`) happened to match the `agent_id`. That is no longer the case. 

However, promotions are scoped to projects/agents, so `{{agent_id}}` is the correct variable to use here.

Reply:
I have fixed this, still see there are some issues with config, any more pointers?